src/main/java/org/broadinstitute/hellbender/tools/funcotator/dataSources/gencode/GencodeFuncotationFactory.java

```diff
@@ -717,28 +910,30 @@ else if ( exon.getStopCodon() != null ) {
                                                                                 final Allele altAllele,
                                                                                 final GencodeFuncotation.VariantType variantType,
                                                                                 final GencodeGtfExonFeature exon,
+                                                                                final int numberOfExonsInTranscript,
                                                                                 final SequenceComparison sequenceComparison ){
 
         Utils.nonNull(variant);
         Utils.nonNull(altAllele);
         Utils.nonNull(variantType);
         Utils.nonNull(exon);
+        ParamUtils.isPositive(numberOfExonsInTranscript, "Number of exons in transcript must be positive (given: " +numberOfExonsInTranscript + ")");
         Utils.nonNull(sequenceComparison);
 
-        // Get our start position:
-        final int startPos = sequenceComparison.getAlleleStart();
-
-        // Determine end position based on whichever allele is longer:
-        final int endPos;
-        if ( altAllele.length() >= variant.getReference().length()  ) {
-            endPos = sequenceComparison.getAlleleStart() + altAllele.length() - 1;
-        }
-        else {
-            endPos = sequenceComparison.getAlleleStart() + variant.getReference().length() - 1;
-        }
-
-        // Calculate the number of inserted bases so we can account for them in the splice site calculations:
-        final int numInsertedBases = (altAllele.length() > variant.getReference().length()) ? altAllele.length() - variant.getReference().length() : 0;
+//        // Get our start position:
```

Comment:
Can you delete code that is being commented out?

Reply:
Sure.